src/github.com/getlantern/flashlight/client/balancer.go

```diff
@@ -37,7 +38,10 @@ func (client *Client) initBalancer(cfg *ClientConfig) (*balancer.Balancer, error
 			log.Debugf("Adding chained server: %v", s.Addr)
 			dialers = append(dialers, dialer)
 		} else {
-			log.Errorf("Unable to configure chained server. Received error: %v", err)
+			elog.Log(err, errlog.WithOp("configure"), errlog.WithProxy(&errlog.ProxyingInfo{
```

Comment:
@uaalto Never mind, I understand now.  I had forgotten the context of the comment :)  Yes, I think the convenience method approach is good.